extensions/web-tracker/wab/src/main/resources/META-INF/cxs/schemas/identify/properties.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "https://unomi.apache.org/schemas/json/events/identify/properties/1-0-0",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "self": {
+    "vendor": "org.apache.unomi",
+    "target": "events",
+    "name": "identifyProperties",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "title": "IdentifyProperties",
+  "type": "object",
+  "properties": {
+    "nickname": {
+      "type": "string"
+    },
+    "favoriteCompiler": {
+      "type": "string"
+    },
+    "industry": {
+      "type": "string"
+    }
+  }
```

Comment:
Careful, this schema doesn't contains "unevaluatedProperty:false", so it could produce mapping explosion